README.md

```diff
@@ -1,5 +1,145 @@
 
-# Specification 'origin-policy'
+# Explainer: Origin Manifest
 
-This is the repository for origin-policy. You're welcome to contribute! Let's make the Web rock our socks
-off!
+**tl;dr** Origin Manifest (formerly Origin Policy) is a web platform mechanism
+that aims to shift configuring the origin from web applications to the origin
+itself. For this the origin provides a manifest file in a predefined well-known
+location which browsers can load and apply.
+
+**Note:** Though the current draft still uses the name Origin Policy, we use the
+new term Origin Manifest here to avoid confusion with the Same Origin Policy.
+
+
+## The Problems and Goals
+
+### Configurations with origin-wide effects
+**Problem:** Some configurations, e.g. like HTTP headers, affect an entire
+  origin even though they might have been set through only a sub-resource load.
+  Misconfiguration of font resource header can have dramatic effects for the
+  origin, e.g. a wrong HPKP value.
+
+**Goals:** Configurations for web applications with effects on entire origin in
+  a more structured way to minimize the chances for origin misconfiguration
+  through individual web applications sharing the same origin.
+
+### HTTP header redundancy
+**Problem 1:** Same HTTP headers have to be sent again and again without the
+  values actually to change. Sometimes these headers add significant number of
+  bytes, e.g. CSP.
```

Comment:
Nits: s/same/some/, s/without the values actually to change/without actually changing the values/ (or "Some HTTP headers are sent with every response, even though the value is more or less constant for resources across the entire origin.")